internal/batches/executor/coordinator.go

```diff
@@ -221,6 +224,18 @@ func (c *Coordinator) loadCachedStepResults(ctx context.Context, task *Task, glo
 	return nil
 }
 
+func writeToCache(ctx context.Context, cache cache.Cache, stepResult execution.AfterStepResult, task *Task) error {
```

Comment:
We now have a writeToCache and a (*Coordinator).writeCache method, can we rename both to make clearer what each are?

Maybe `writeToCache` => `writeStepCacheResult`, `(*Coordinator).writeCache` => `(*Coordinator).writeExecutionCacheResult`?

Reply:
Done.